### ISSUE TITLE
Feature request: date format other than IFTTT's one a.k.a. working with Zapier

### ISSUE THREAD

**Issue by sanzoghenzo** [bachya/Sifttter-Redux#18]
Hi all,
I just started using Zapier since I found it has a mail parser that helps me extract useful info from emails.
I already set it to parse mail alerts from my bank when I withdraw money from ATM, and use date, time and amount to create a sifttter parseable file. anyway, the date is in form "DD/MM/YY" (and the time in 24hr, but it's not a problem), and Sifttter cannot handle that.
Is it possible to add support to this date format? 
basically my dropbox entry looks like that:

```
@begin
@date DD/MM/YY
* HH:mm ATM withdraw X Eur
@end
```

I think adding this functionality will result in more flexibility and support for even more services/scripts.
